src/drivers/github.js

```diff
@@ -18,6 +18,12 @@ const {
   GITHUB_EVENT_NAME
 } = process.env;
 
+const branch_name = (branch) => {
+  if (!branch) return;
+
+  return branch.replace('refs/heads/', '');
```

Comment:
does we need logic for `refs/tags/` too?

Reply:
👁️  💯  Probably yes, users may trigger the workflow on tag events.

Reply:
```suggestion
  return branch.replace(/refs\/(head|tag)s\//, '');
```

Reply:
For security I guess so, I dont see when that would happen 🤔 but definitely is secure